hack/build-go.sh

```diff
@@ -40,7 +40,7 @@ if [ "$GO111MODULE" == "off" ]; then
 	export GO15VENDOREXPERIMENT=1
 	export GOBIN=${PWD}/bin
 	export GOPATH=${PWD}/gopath
-	go install -tags no_openssl -ldflags "${LDFLAGS}" "$@" ${REPO_PATH}/multus
+	go build -o ${PWD}/bin/multus -tags no_openssl -ldflags "${LDFLAGS}" "$@" ${REPO_PATH}/cmd
```

Comment:
Here's what I needed to change that was causing the GitHub actions build of this to fail. It's kind of redundant with the `export GOBIN` above, but should wind up with the binary in the correct place regardless.